### PR TITLE
MGMT-9166 - Help text disappears once there is an error

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-9166

**Some examples of input fields with helper and error texts:**

image
image
image
image
image
image
image

### CODE REVIEW

**Comment by jgyselov** [comment]
@nirfarkas PTAL

**Review by ammont82 — APPROVED** [review]

**Review by ammont82 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "openshift-assisted-ui-lib",
-  "version": "0.0.0",
+  "version": "2.16.3",
```
Comment (rawagner) [inline package.json:3]: changing the version is a mistake, right ?

File: src/common/components/ui/formik/InputField.tsx [hunk 2]
```diff
@@ -24,7 +32,7 @@ const InputField: React.FC<
       inputError,
       description,
       labelInfo,
-      showErrorMessage = true,
+      // showErrorMessage = true,
```
Comment (rawagner) [inline src/common/components/ui/formik/InputField.tsx:35]: there is couple of places where this prop is still used. I think if we change
```
{errorMessage && (
  <HelperText fieldId={fieldId} isError>
    {errorMessage}
  </HelperText>
)}
```

to
```
{errorMessage && showErrorMessage && (
   <HelperText fieldId={fieldId} isError>
     {errorMessage}
   </HelperText>
)}
```
we should be good

**Review by rawagner — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jgyselov) [reply package.json]: Yes.

**Review by ammont82 — APPROVED** [review]

**Review by rawagner — APPROVED** [review]

**Pull request merged**